### PR TITLE
Move fidesapi -> fidesctl/api

### PR DESCRIPTION
**Pull request by ThomasLaPiana**
Closes <issue>

### Code Changes

* [x] move the files
* [x] update imports
* [x] update docs
* [x] update additional infra-type files
* [x] update `setup.py` to remove `package_data` as it is redundant with our `MANIFEST.IN` (docs [here](https://packaging.python.org/en/latest/guides/using-manifest-in/))

### Steps to Confirm

* [ ] automated tests pass
* [ ] any poking and prodding you can think of 🤷 
* [ ] run `nox -s "build(test)"` and confirm that the `alembic.ini` file is in there

### Pre-Merge Checklist

* [x] All CI Pipelines Succeeded
* Documentation Updated:
  - [x] documentation complete, or draft/outline provided (tag docs-team to complete/review on this branch)
  - [ ] documentation issue created (tag docs-team to complete issue separately)
* [x] Issue Requirements are Met
* [x] Update `CHANGELOG.md`

### Description Of Changes

This PR fixes a very weird idiosyncrasy we've had in our codebase where `fidesapi` and `fidesctl` get loaded as separate modules when a user does `pip install fidesctl` (two for the price of one, but weird behavior nonetheless)

This should also be merged after #881, otherwise trying to install as a standalone will break things now that the api is part of the main package and requires `fastapi` to not fail the import

### CODE REVIEW

**Comment by ThomasLaPiana** [comment]
reminder to self to not merge until after #881 gets merged

**Comment by ThomasLaPiana** [comment]
I'm revisiting this to fix the merge conflicts

File: MANIFEST.in
Comment (ThomasLaPiana) [inline MANIFEST.in:2]: this gets included by default (docs [here](https://packaging.python.org/en/latest/guides/using-manifest-in/))

**Review by earmenda — APPROVED** [review]
lgtm

**Pull request merged**
